bundles/config/org.eclipse.smarthome.config.core/OSGI-INF/I18nConfigOptionsProvider.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
How about using the nice DS annotations that you have introduced recently? Adding further XMLs only makes the refactoring harder in future!